docs/modules/operation/pages/deep-dive/service-assurance/downtime-model.adoc

```diff
@@ -22,32 +22,14 @@ The scheduler changed the polling interval back to five minutes.
 <downtime interval="30000" begin="0" end="300000" /><!-- 30s, 0, 5m --> <1>
 <downtime interval="300000" begin="300000" end="43200000" /><!-- 5m, 5m, 12h --> <2>
 <downtime interval="600000" begin="43200000" end="432000000" /><!-- 10m, 12h, 5d --> <3>
-<downtime interval="3600000" begin="432000000" delete="never"/><!-- 1h, 5d --> <4>
 ----
 
 <1> From 0 seconds after an outage is detected until 5 minutes, set the polling interval to 30 seconds.
 <2> After 5 minutes of an ongoing outage until 12 hours, set the polling interval to 5 minutes.
 <3> After 12 hours of an ongoing outage until 5 days, set the polling interval to 10 minutes.
-<4> After 5 days of an ongoing outage, the service will be polled only once per hour.
-Services are not deleted during this time.
-
-The last downtime model can have the `delete` attribute; this lets you influence the service lifecycle.
-It defines the intended behavior if a service does not come back online after the specified period of time.
-The following attributes can modify `delete`:
-
-[options="autowidth"]
-|===
-| Value | Description
 
-| never
-| Services will never be deleted automatically (default value).
-
-| managed
-| Only managed services will be deleted.
-
-| always
-| Managed and unmanaged services will be deleted.
+Note that after five days of an ongoing outage, the service will be polled only once per hour.
+Services are not deleted during this time.
 
-| not set
-| Assumes a default value of `never`.
-|===
+//what status do services have? A and N? What does that stand for?
+// does editing a model affect current outages or only new ones?
```

Comment:
The downtime model doesn't affect outages - it affects the scheduling of poller tasks.  It's affected by outages (and doesn't know the difference between new/existing)
```suggestion
```